desktop_native/src/password/macos.rs

```diff
@@ -0,0 +1,43 @@
+use anyhow::Result;
+use security_framework::passwords::{
+    delete_generic_password, get_generic_password, set_generic_password,
+};
+
+pub async fn get_password<'a>(service: &str, account: &str) -> Result<String> {
```

Comment:
I think all these methods are blocking, does it do anything to make these methods async?

Reply:
Keeping the same pattern as windows? Or can you mix async and non async calls?

Reply:
Removed async.